src/otx/recipes/stages/instance-segmentation/incremental.py

```diff
@@ -14,7 +14,7 @@
 
 runner = dict(max_epochs=300)
 
-optimizer_config = dict(_delete_=True, grad_clip=dict(max_norm=35, norm_type=2))
+optimizer_config = dict(_delete_=True)
```

Comment:
Does eliminating the `grad_clip` also be decided by experimental results? In my previous experience, grad_clip is much useful especially in very small dataset cases (i.e. total training images < 16)

Reply:
Similar question. Why are you changing incremental learning recipe in Semi-SL PR?

Reply:
I will revert it back, sure. I just found no difference using it or not. But I didn't try 16 images